pkg/apis/bundle/v1alpha1/cluster_bundle.proto

```diff
@@ -50,15 +52,15 @@ message ClusterBundleSpec {
   // the bundle, then the version string must be incremented.
   //
   // If a bundle is versioned, then all its components must be versioned.
-  string version = 1;
+  string version = 2;
 
   // Kubernetes objects grouped into component packages and versioned together.
   // These could be applications or they could be some sort of supporting
   // object collection.
-  repeated ComponentPackage components = 2;
+  repeated ComponentPackage components = 3;
```

Comment:
Isn't our plan to make this a reference?

Reply:
Yes, but this is a massive change; and might not make Alpha 1.X given the size.

Reply:
I started doing it in this PR, but then reverted it.

Reply:
Yes, we discussed this being a name and version that reference a separate component CRD.

Reply:
Let me be more clear: this is definitely coming, but I wanted to incrementally approach that.